tide/master/MasterWindow.cpp

```diff
@@ -317,9 +317,9 @@ void MasterWindow::_setupMasterWindowUI()
 
     // Forward background touch events
     connect( _displayGroupView, &DisplayGroupView::backgroundTap,
-             this, &MasterWindow::hideDock );
+             this, &MasterWindow::hideLauncher );
```

Comment:
removing the old signal(s) is part of the follow-up commit removing the entire old dock?